airflow-core/src/airflow/ui/public/i18n/locales/pl/common.json

```
@@ -65,7 +65,8 @@
     "runAfter": "Wykonaj po",
     "runType": "Typ wykonania",
     "sourceAssetEvent": "Zdarzenie źródłowego zasobu",
-    "triggeredBy": "Uruchomiony przez"
+    "triggeredBy": "Uruchomiony jako",
```

```suggestion
    "triggeredBy": "Uruchomiony przez",
```
I think previous one is clearer

---

If I am not mistaken- "triggered by" is about "schedule / manual" not about "person". so I thought "przez" reflects it better.

---

```
export type DagRunTriggeredByType = 'cli' | 'operator' | 'rest_api' | 'ui' | 'test' | 'timetable' | 'asset' | 'backfill';
```

---

* Uruchomiony przez adam
* Urichominoy jako : cli 

? WDYT ?

---

Merging for now. We can change it later. I think we will see it when those reasons will be translated (they are not now).